ScnSoft-Education/LeetCodeTraining.Tests/LongestSubstringTest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LeetCodeTraining.LongestSubstring1;
+
+namespace LeetCodeTraining.Tests
+{
+    public class LongestSubstringSolutionTest
+    {
+        [Fact]

```

Comment:
Please try to use [Theory] and [InlineData] attributes for the xUnit test and rewrite it to get rid of duplicated code in the test.
Hint: in case of using suggested attributes you will have 4 attributes InlineData before the test and don't have Arrange block in the test, actually arrange block will be moved to in test's parameters.
https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/